tests/unit/test_accelerator_classes.py

```diff
@@ -27,6 +30,42 @@ def test_lhc_adt_b2():
     _check_exciter_bpm_detection(accel, "X", "BPMWA.B5R4.B2", "BPMWA.A5R4.B2")
     _check_exciter_bpm_detection(accel, "Y", "BPMWA.B5L4.B2", "BPMWA.A5L4.B2")
 
+@pytest.mark.basic
+def test_psbase_best_knowledge():
+
+    accel = Ps(
+            year="2021",
+            scenario="lhc",
+            cycle_point="0_injection",
+            str_file="ps_inj_lhc.str",
+            tune_method="qf"
+            )
+
+    with pytest.raises(AttributeError):
+        _ = accel.get_base_madx_script(best_knowledge=True)
```

Comment:
Let's maybe add a comment above this to explain why we expect to raise and error? I'm not familiar enough with the PS to determine from the test.